quickstack/manifests/controller_common.pp

```diff
@@ -152,11 +152,19 @@
   $ceph_enpoints                 = $quickstack::params::ceph_endpoints,
   $ceph_user                     = $quickstack::params::ceph_user,
   $ceph_vlan                     = $quickstack::params::ceph_vlan,
+  $sensu_client_enable           = $quickstack::params::sensu_client_enable,
   $sensu_rabbitmq_host           = $quickstack::params::sensu_rabbitmq_host,
   $sensu_rabbitmq_user           = $quickstack::params::sensu_rabbitmq_user,
   $sensu_rabbitmq_password       = $quickstack::params::sensu_rabbitmq_password,
-  $sensu_client_subscriptions_controller   = 'moc-sensu',
-  $sensu_client_keepalive       = { "thresholds" => { "warning" => 60, "critical" => 300 }, "handlers" => ["node-email"], "refresh" => 3600 },
+  $sensu_client_subscriptions    = $quickstack::params::sensu_subscriptions_controller,
```

Comment:
@ravisantoshgudimetla right now everything points to moc-sensu, this is prep work for trying to organize the subscriptions more in the future.